### PR TITLE
#858 add chat action constants

### PR DESCRIPTION
**Pull request by smoqadam**
#858 add chat action constants

### CODE REVIEW

File: src/Telegram.php [hunk 1]
```diff
@@ -25,6 +25,57 @@
...
+
+    /**
+     * Record Video Note chat action
+     */
+    const ACTION_RECORD_VIDEO_NOTE = 'record_video_note';
+
+    /**
+     * Upload Video note chat action
+     */
+    const ACTION_UPLOAD_VIDEO_NOTE = 'record_video_note';
```
Comment (noplanman) [inline src/Telegram.php]: @smoqadam 

> ...to prevent from future mistakes like typos.

Unless they are built-in :grin:
Please correct this one.

**Comment by noplanman** [comment]
@php-telegram-bot/developers 
Would we want these all in the `Telegram` class or a separate class `Action` that has just those constants in it, i.e.
```php
class Action
{
    const TYPING            = 'typing';
    const UPLOAD_PHOTO      = 'upload_photo';
    const RECORD_VIDEO      = 'record_video';
    const UPLOAD_VIDEO      = 'upload_video';
    const RECORD_AUDIO      = 'record_audio';
    const UPLOAD_AUDIO      = 'upload_audio';
    const UPLOAD_DOCUMENT   = 'upload_document';
    const FIND_LOCATION     = 'find_location';
    const RECORD_VIDEO_NOTE = 'record_video_note';
    const UPLOAD_VIDEO_NOTE = 'upload_video_note';
}
```

We'd also have to decide what namespace to put them in.

**Comment by akalongman** [comment]
+1 for separate class

**Comment by smoqadam** [comment]
`ChatAction` class wouldn't be more meaningful? 

```php
class ChatAction
{
    /**
     * Typing chat action
     */
    const TYPING = 'typing';

    /**
     * Upload Photo chat action
     */
    const UPLOAD_PHOTO = 'upload_photo';

    /**
     * Record Video chat action
     */
    const RECORD_VIDEO = 'record_video';
    .....
```

#### usage

```php
Request::sendChatAction([
    'chat_id' => $chat_id,
    'action'  => Longman\TelegramBot\ChatAction::TYPING,
]);
```

**Comment by noplanman** [comment]
> ChatAction class wouldn't be more meaningful?

Absolutely! Thanks for the fixed PR :+1:

**Pull request merged**